scss/mixins/_list-group.scss

```diff
@@ -1,7 +1,10 @@
+@include deprecate("`list-group-item-variant()`", "v5.3.0", "v6.0.0");
+
+
 // List Groups
 
 // scss-docs-start list-group-mixin
-@mixin list-group-item-variant($state, $background, $color) {
+@mixin list-group-item-variant($state, $background, $color, $background-hover: shade-color($background, 10%)) {
```

Comment:
I don't think we need to keep this modification (and line 16) since this mixin is now deprecated.